test/utils/simple_server.js

```diff
@@ -4,7 +4,17 @@ const http = require('http');
 const P = require('bluebird');
 
 module.exports = (options) => {
-    const server = http.createServer((req, res) => res.end());
+    this.hitcounter = new options.prometheus.Counter({
+        name: 'hitcount',
+        help: 'a hit counter',
+        labelNames: ['worker_id']
+    });
+
+    const server = http.createServer((req, res) => {
+        this.hitcounter.labels(options.config.worker_id).inc();
+        options.metrics.increment(`${options.config.worker_id}.hitcount`);
+        res.end('ok\n');
+    });
```

Comment:
I guess since it's quite preductable with a counter, we can actually write a test. Start a server, call the endpoint several times, fetch the counter, see what we've got.

Reply:
Added a test.  I think it is sufficient, but let me know if it isn't.